docs/code/cloud_storage.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+
+# pylint: disable=wrong-import-position
+# pylint: disable=wrong-import-order
+# pylint: disable=pointless-string-statement
+# pylint: disable=no-name-in-module
+# pylint: disable=no-value-for-parameter
+# type: ignore[arg-type]
```

Comment:
why ignore mypy and flake8 in the whole file?

Reply:
mypy will raise `Module 'tensorbay.dataset' has no attribute 'AuthData'` when tensorbay before 1.7

I will delete flake8 ignore.